gfx_scene.cpp

```diff
@@ -1016,7 +1310,7 @@ class GfxSceneInternal
                 if(gfxSceneImport(scene, image_file.c_str()) != kGfxResult_NoError)
                     continue; // unable to load image file
                 image_ref          = gfxSceneFindObjectByAssetFile<GfxImage>(scene, image_file.c_str());
-                images[gltf_image] = image_ref;
+                images[gltf_texture.image] = image_ref;
```

Comment:
Theres a slight change as now textures are looked up based on the base image pointer. This simplifies the code